courses/machine_learning/intro_to_tensorflow/2_dataset_api.ipynb

```diff
@@ -0,0 +1,742 @@
+{
```

Comment:
<p>suggest that you compute the average value of the columns in the dataset before you get to regression.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Added in the previous lab the baseline model that outputs the mean and computed the test error for both the linear model and the one predicting the mean . Since this is lab is a repetition of the previous lab  (same models &amp; same data), but we are only using tf.data API instead of arrays, I am skipping this step in this lab. </p>